addon/components/materialize-copyright.js

```diff
@@ -0,0 +1,25 @@
+import Ember from 'ember';
+import layout from '../templates/components/materialize-copyright';
+
+export default Ember.Component.extend({
+  init: function() {
+    this._super();
+    if (this.get('startYear') != null) {
+      Ember.assert('Property startYear must be less than or equal to the current year.',
+        this.get('startYear') <= new Date().getFullYear());
+    }
+  },
+  layout: layout,
+  classNames: ['footer-copyright'],
+  startYear: null,
+  text: null,
+  date: function () {
+    var returnDate;
+    if (this.get('startYear') === null || this.get('startYear') === new Date().getFullYear()) {
+      returnDate = new Date().getFullYear();
+    } else {
+      returnDate = this.get('startYear') + ' - ' + new Date().getFullYear();
+    }
+    return returnDate;
+  }.property()
```

Comment:
We have recently disabled prototype extensions, so please change this to 
```js
date: Ember.computed(function () { ... }),
```
the body of the function should remain the same